src/apps/interface.ts

```diff
@@ -24,11 +30,84 @@ export interface MSafeAppHelper<T> {
     txType: TransactionType;
     txSubType: string;
     intentionData: T;
-    suiClient: SuiClient;
+    clientUrl: string;
+    account: WalletAccount;
+  }): Promise<TransactionBlock>;
+}
+
+export interface IAppHelperLegacy<T> {
```

Comment:
Suggest rename to IAppHelperInternalLegacy, and add some comment for third party instruction.